ui/app/models/transit-key.js

```diff
@@ -43,7 +43,7 @@ const ACTION_VALUES = {
 
 export default DS.Model.extend({
   type: attr('string', {
-    defaultValue: 'aes256-gcm96',
+    defaultValue: 'aes128-gcm96',
```

Comment:
curious why we're lowing the bit count here (assuming the 256 stands for bits).  I'm also guessing, the higher the bit count the more secure?

Reply:
great question! in this case we're only changing the default value so it automatically selects the first item in the dropdown. the other option is still available. this way we're listing all options in alphabetic and numeric order:

![image](https://user-images.githubusercontent.com/903288/87814644-f5fd2600-c818-11ea-9a2a-ef8d154b107c.png)